ui/ui-components/components/model/overview/EntityInfoContainer.tsx

```diff
@@ -0,0 +1,20 @@
+import { Media } from "react-bootstrap";
+import { Models } from "../../../utils/apiData";
+import AppIcon from "../../AppIcon";
+
+interface Props {
+  app?: Models.AppType;
+}
+
+const EntityInfoContainer: React.FC<Props> = ({ app, children }) => {
+  return app && app.icon ? (
+    <Media>
+      <AppIcon src={app.icon} size={42} className="mr-3" />
+      <Media.Body>{children}</Media.Body>
+    </Media>
+  ) : (
+    <>{children}</>
+  );
+};
+
+export default EntityInfoContainer;
```

Comment:
Still working on this name. An "entity" is a a Grouparoo object such as a source, destination, group, or grouparoo model. I could also call it a "GrouparooObjectContainer". I didn't want to call it model because of the already overloaded use of the term.